### ISSUE TITLE
Survival plot with breaks option error

### ISSUE THREAD

**Issue by sethfrnd** [alexpkeil1/qgcomp#26]
When breaks are specified the plot function for qgcomp.cox.boot results in an error:

Error in 0:(x$q - 1) : argument of length 0 

Is this because the quartiles are set to NULL?

**Comment by alexpkeil1** [comment]
This was somewhat purposeful since the values of "breaks" are not guaranteed to be quantiles but they are labeled that way in the plots, but the simple workaround given below won't cause any ill side-effects here since estimates/variance/confidence intervals do not hinge on what the breaks represent.

```library(qgcomp)
library(ggplot2)
set.seed(50)
N=200
dat <- data.frame(time=(tmg <- pmin(.1,rweibull(N, 10, 0.1))), 
                  d=1.0*(tmg<0.1), x1=runif(N), x2=runif(N))
expnms=paste0("x", 1:2)
f = survival::Surv(time, d)~x1 + x2


(obj1 <- qgcomp.cox.boot(f, expnms = expnms, data = dat, q=4, B=10, MCsize=2000))
(obj1b <- qgcomp.cox.boot(f, expnms = expnms, data = dat, breaks=obj1$breaks, q=NULL, B=10, MCsize=2000))


# not run
plot(obj1) # 
# not run
#plot(obj1b) # gives error
#workaround (manually set q)
obj2 = obj1b
obj2$q = length(obj1$breaks[[1]])-1
plot(obj2) # works
all.equal(obj1b, obj2) # notes only difference is q
```

**Comment by alexpkeil1** [comment]
Leaving this issue open - this should result in a warning to the user rather than an error.

**Comment by sethfrnd** [comment]
This works great! But yes it does return an Error not a warning.

**Comment by alexpkeil1** [comment]
Fixed in v2.9.0 on CRAN
